public/assets/config.yml.dist

```diff
@@ -1,6 +1,7 @@
 ---
 # Homepage configuration
 # See https://fontawesome.com/icons for icons options
+# Use version 5.15.4
```

Comment:
Perhaps instead of adding an extra line, could you change the url to point to the v5 list?

https://fontawesome.com/v5/search

Reply:
I accidentally deleted the 2 old patches without know it will also remove the pull request. At lao to last one didn't change the documentations and docker rework. So I made 2 pulls #404 #405. Change on main branch and docker-rework branch